modules/node-observability-scripting.adoc

```diff
@@ -0,0 +1,108 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/using-node-observability-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="node-observability-scripting_{context}"]
+= Node Observability Operator Scripting
+
+[About Scripting]
+
+.Prerequisites
+
+* You have installed the Node Observability Operator.
+* You have created the `NodeObservability` custom resource (CR).
+* You have access to the cluster with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a `NodeObservabilityScripting` resource file named `nodeobservabilityrun-script.yaml` that contains the following text:
```

Comment:
@subhtk - sorry only spotted this now. Could you please change `NodeObservabilityScripting` to `NodeObservabilityRun`